extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -1335,6 +1335,29 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 			this.wizard.message = { text: '' };
 		}
 
+		const errorDetails: string[] = [];
+		if (this._windowsUserAccountText.value === undefined || this._windowsUserAccountText.value === '') {
```

Comment:
it is good that are adding some validation, but should not we have character set validation. Like Username, Storage Account name and Container name we have defined set of characters. Special correctors are not supported.  You may quickly check with PM.